mystery_o_matic/clues.py

```
@@ -200,6 +235,36 @@ def string_spanish(self):
         s += f'a {object} cuando llegué a {self.place} a las {self.time}"'
         return s
 
+    def string_russian(self):
+        object = self.object
+        r = randint(0, 2)
```

Ruff (0.15.5)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)